### ISSUE TITLE
MFMailComposeViewController

### ISSUE THREAD

**Issue by dusty-commits** [vhesener/Closures#15]

**Comment by Overcot** [comment]
Can I try doing it?

**Comment by dusty-commits** [comment]
Yes, of course. Take a look at the notes on contributing and submit a PR when ready https://github.com/vhesener/Closures/blob/master/CONTRIBUTING.md

**Comment by dusty-commits** [comment]
Feel free to send a link showing incremental progress if you want some early feedback before submitting. Thanks

**Comment by Overcot** [comment]
@vhesener Submitted my pr #56 to show incremental progress, want to get some early feedback
